doc/whats_new.rst

```diff
@@ -93,6 +93,8 @@ API
 
 - Prepare transition to Python 3. This release will be the last release compatible with Python 2. The next version will be Python 3 only.
 
+- func:`mne.viz.plot_evoked_topo` and other functions requiring a color cycle will now default to Matplotlib rcParams colors through :func:`mne.viz.utils._get_color_list`: by `Stefan Appelhoff`_
```

Comment:
only things in `python_reference.rst` will link properly with `:func:`, also we don't usually talk about private functions in `whats_new.rst`. So instead of having the private functions mentioned it's better to simply omit the "through" part or say something like "... and other functions requiring a color cycle have been refactored to ..." or